keps/0020-manual-plan-execution.md

```diff
@@ -9,7 +9,7 @@ owners:
   - "@zen-dog"
 creation-date: 2019-11-08
 last-updated: 2020-01-20
```

Comment:
lets capture the `last-updated` so we can track when it was implemented in history.